_data/scala-compiler-team.yml

```
@@ -0,0 +1,104 @@
+# - name: Core Team
+#   description: Knows everything
+#   image: /resources/img/frontpage/scala-spiral.png
+#   city: Lausanne, Switzerland
+#   email: scala.center@epfl.ch
+#   twitter: scala_lang
+#   github: scala
+#   linkedin: https://www.linkedin.com/in/janedoe/
+#   mastodon: https://fosstodon.org/@janedoe
+#   meetup: https://www.meetup.com/your/url/here
+
+- name: Martin Odersky
+  description: Language Creator
+  github: odersky
+
+- name: Piotr Chabelski
+  description: Product Manager
```

NIT likewise, not sure if necessary to list it all
```suggestion
  description: Product Manager, Tooling Engineer, Specialised Engineer (Scala CLI)
```

---

I think maybe drop `Tooling Engineer` ? It's implied with Scala CLI :D I tried to keep it always at 2 max if possible